knownprojects_build/_sql/_procedures.sql

```diff
@@ -0,0 +1,168 @@
+/* 
```

Comment:
the concern here is that if we create this procedure at this step, `_combined` doesn't exist yet, will a new postgres database throw an error for creating a procedure on a table that doesn't exist?

Reply:
Good point. We can move them either to the script where they get called, or to a procedure declaration script that gets called later in the process.